k8s/service_controller/Makefile

```diff
@@ -14,10 +14,10 @@ binary: $(EXE)
 $(EXE): $(SRC_FILES)
 ifeq (,$(shell which go))
 	@$(MAKE) --no-print-directory binary-docker
+	docker build -t $(IMAGE):latest .
 else
 	@$(MAKE) --no-print-directory binary-local
 endif
-	docker build -t $(IMAGE):latest .
```

Comment:
Even on a local build I still want to build the image since at some point I expect it to be run via docker in the tests.